source/02-FN/02.ptx

```diff
@@ -12,6 +12,499 @@
+        
+        <task>
+          <p><m>y=2x+14</m></p>
+          <answer>
+            <p>
+              <m>f(x)=2x+14</m>
+            </p>
+          </answer>
+        </task>
+
+        <task>
+          <p><m>y+x=3x^2-5</m></p>
+          <answer>
+            <p>
+              <m>f(x)=3x^2-x-5</m>
+            </p>
+          </answer>
+        </task>
+  
+        <task>
+          <p><m>\displaystyle \frac{2}{x}-x^4 = y-5          </m></p>
+          <answer>
+            <p>
+              <m>f(x)=\displaystyle \frac{2}{x}-x^4 +5  </m>
+            </p>
+          </answer>
+        </task>
+        
+  
+        
+  
+        
+        
+      </activity>
+
+      <activity xml:id="activity-fn-table">
+        <statement>
+          <p> Let <m>f(x)=3x^2-4x+1</m>. Find the value of <m>f(x)</m> for the given values of <m>x</m>.
+        </p>
+        
+        <table>
+          <tabular>
+            <row header="yes" bottom="minor" >
+              <cell><m>x</m></cell> 
+              <cell><m>f(x)</m></cell>
+            </row>
+            <row halign="center">
+              <cell><m>-5</m></cell>
+            </row>
+            <row halign="center">
+              <cell><m>-\frac{1}{2}</m></cell>
+            </row>
+            <row halign="center">
+              <cell><m>0</m></cell>
+            </row>
+            <row halign="center">
+              <cell><m>2</m></cell>
+            </row>
+            <row halign="center">
+              <cell><m>10</m></cell>
+            </row>
+          </tabular>
+        </table>  
+      </statement>
+
+        <answer>
+          <p>
+            <table>
+              <tabular>
+                <row header="yes" bottom="minor" >
+                  <cell><m>x</m></cell> 
+                  <cell><m>f(x)</m></cell>
+                </row>
+                <row halign="center">
+                  <cell><m>-5</m></cell>
+                  <cell><m>96</m></cell>
+                </row>
+                <row halign="center">
+                  <cell><m>-\frac{1}{2}</m></cell>
+                  <cell><m>\frac{15}{4}</m></cell>
+                </row>
+                <row halign="center">
+                  <cell><m>0</m></cell><cell><m>1</m></cell>
+                </row>
+                <row halign="center">
+                  <cell><m>2</m></cell><cell><m>5</m></cell>
+                </row>
+                <row halign="center">
+                  <cell><m>10</m></cell><cell><m>261</m></cell>
+                </row>
+              </tabular>
+            </table> 
+          </p>
+        </answer>
+      </activity>
+
+      <remark>
+      If we are asked to find the value of <m>f(x)</m> for a certain <m>x</m>-value, say <m>x=5</m>, we use the notation <m>f(5)</m> to indicate that.
+      </remark>
+
+      <activity xml:id="activity-fn-values">
+        <introduction>
+          <p> Let <m>f(x)</m>, <m>g(x)</m>, and <m>h(x)</m> be defined as shown. 
+            <md>
```

Comment:
@AbbyANoble Use a `<md>` like this to line up the equals signs, instead of separate `<me>`s